ceph_deploy/tests/test_cli_mon.py

```diff
@@ -65,11 +63,11 @@ def test_simple(tmpdir, capsys):
 """)
 
     ns = argparse.Namespace()
-    ns.pushy = mock.Mock()
-    conn = mock.NonCallableMock(name='PushyClient')
+    ns.pushy = Mock()
```

Comment:
this is a red flag: we are now maintaining tests for libraries that we don't even use anymore (re: pushy)

This test should be removed